test/e2e/deploy-ingress-nginx.yaml

```diff
@@ -0,0 +1,674 @@
+apiVersion: v1
```

Comment:
why not just a simple nginx deployment ratehr than the whole ingress watcher? This file is kind of huge. Was hoping just a deployment. If you have reasons for the whole ingress controller we should talk about them somewere in the e2e